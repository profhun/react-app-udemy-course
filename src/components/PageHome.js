import React from 'react';
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'

const PageHome = (props) => (
    <div>
        <h1>
            Hoooome
        </h1>
        <ExpenseListFilters />
        <ExpenseList {...props} />
    </div>

);

export default PageHome;