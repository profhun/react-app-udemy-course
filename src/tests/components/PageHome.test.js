import React from 'react';
import { shallow } from 'enzyme';
import PageHome from '../../components/PageHome';

test('render/component PageHome', () => {

    const wrapper = shallow(<PageHome />);
    expect(wrapper.find('h1').text()).toBe("Hoooome");
    expect(wrapper).toMatchSnapshot();
});