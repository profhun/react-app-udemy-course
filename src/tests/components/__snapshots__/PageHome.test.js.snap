// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render/component PageHome 1`] = `
<div>
  <h1>
    Hoooome
  </h1>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
